doc/source/raster_vector_point.md

```diff
@@ -0,0 +1,191 @@
+
+```{code-cell} ipython3
+:tags: [remove-cell]
+
+# To get a good resolution for displayed figures
+from matplotlib import pyplot
+pyplot.rcParams['figure.dpi'] = 600
+pyplot.rcParams['savefig.dpi'] = 600
+pyplot.rcParams['font.size'] = 9
+```
+
+## Raster–vector operations
+
+### Rasterize
+
+{func}`geoutils.Vector.rasterize`
+
+Rasterization of a vector is **an operation that allows to translate some information of the vector data into a raster**, by 
+setting the values raster pixels intersecting a vector geometry feature to that of an attribute of the vector 
+associated to the geometry (e.g., feature ID, area or any other value).
+
+Rasterization generally implies some loss of information, as there is no exact way of representing a vector on a grid. 
+Rather, the choice of which pixels are attributed a value depends on the amount of intersection with the vector 
+geometries and so includes several options (percent of area intersected, all touched, etc).
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+:mystnb:
+:  code_prompt_show: "Show the code for opening example files"
+:  code_prompt_hide: "Hide the code for opening example files"
+
+import matplotlib.pyplot as plt
+import geoutils as gu
+import numpy as np
+
+rast = gu.Raster(gu.examples.get_path("everest_landsat_b4"))
+rast.set_nodata(0)  # Annoying to have to do this here, should we update it in the example?
+vect = gu.Vector(gu.examples.get_path("everest_rgi_outlines"))
+```
+
+```{code-cell} ipython3
+# Rasterize the vector features based on their glacier ID number
+rasterized_vect = vect.rasterize(rast)
+rasterized_vect.plot(ax="new", cmap="viridis")
+```
+
+### Create mask
+
+{func}`geoutils.Vector.create_mask`
+
+Mask creation from a vector **is a rasterization of all vector features that only categorizes geometry intersection as a boolean mask**
+(if any feature falls in a given pixel or not), and is therefore independent of any vector attribute values.
+
+```{code-cell} ipython3
+# Create a boolean mask from all vector features
+mask = vect.create_mask(rast)
+mask.plot(ax="new")
+```
+
+It returns a {class}`~geoutils.Mask`, a georeferenced boolean raster (or optionally, a boolean NumPy array), which 
+can both be used for indexing or index assignment of a raster.
+
+```{code-cell} ipython3
+# Mean of values in the mask
+np.mean(rast[mask])
+```
+
+### Polygonize
+
+{func}`geoutils.Raster.polygonize`
+
+Polygonization of a raster **consists of delimiting contiguous raster pixels with the same target values into vector polygon 
+geometries**. By default, all raster values are used as targets. When using polygonize on a {class}`~geoutils.Mask`, 
+the targets are implicitly the valid values of the mask.
+
+```{code-cell} ipython3
+# Mask 0 values
+rasterized_vect.set_mask(rasterized_vect == 0)
+# Polygonize all non-zero values
+vect_repolygonized = rasterized_vect.polygonize()
+vect_repolygonized.plot(ax="new", column="New_ID")
```

Comment:
Ahah interesting exercise to revert the operation!
But in the example, it's difficult to see the difference with the raster output. Maybe plot only contours, instead of index, so it's clear these are polygons?

Reply:
Good idea, done!